bag_of_words/code/classifiers.py

```diff
@@ -0,0 +1,89 @@
+import pickle, heapq
+from itertools import compress
+import numpy as np
+from sklearn.model_selection import cross_validate
+from sklearn.svm import LinearSVC
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.ensemble import RandomForestClassifier
+
+indir = "../data/processed/"
+
+with open(indir + "bag_of_words.pkl", 'rb') as fin:
+    bow = pickle.load(fin)  # doc-by-word count sparse matrix
+    bow_rownames = pickle.load(fin)  # list of PMIDs

```

Comment:
It's not clear to me how you're getting rownames and colnames separately from this `pickle.load`

Reply:
Calling pickle.load(fin) for 3 times will read in 3 objects in the order that they were written.